packages/property-grid/src/api/WebUtilities.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) 2020 Bentley Systems, Incorporated. All rights reserved.
```

Comment:
need to update the header

Reply:
also this should probably be moved to a helper pkg cause i think i saw this exact file in tree-widget

Reply:
I'll take care of this in a separate PR